### ISSUE TITLE
[Bug] CalculiX crashes for Frequency or Buckling analyses

### ISSUE THREAD

**Issue by donovaly** [FreeCAD/FreeCAD#6579]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Forums discussion

https://forum.freecadweb.org/viewtopic.php?p=579209#p579209

### Version

0.19 (Release)

### Full version info

```shell
OS: Windows 10 Version 2009
Word size of FreeCAD: 64-bit
Version: 0.20.28072 (Git)
Build type: Release
Branch: master
Hash: 08552c6e684aaa325ce8efb6802f1e664a010e4e
Python 3.8.6+, Qt 5.15.2, Coin 4.0.1, OCC 7.5.3
Locale: German/Germany (de_DE)
```

### Subproject(s) affected?

FEM

### Issue description

Follow the steps in https://forum.freecadweb.org/viewtopic.php?p=579212#p579212

result: When using the Windows builds of FC 0.19.3 or weekly builds, Buckling (FC 0.20 only) and Frequency analysis crashes CalculiX.

However, a Win user reported it works, it seems to be only a packaging issue.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by donovaly** [comment]
With the weekly Conds builds, it works. So there is a bug in the LibPack.

**Comment by donovaly** [comment]
OK, some mkl_*.dll files are missing. This affects also the builds for FreeCAD 0.19.3.

I will now compile a new installer for 0.19.4.

All affected users can fix the issue, by putting the content of this ZIP file to the same folder where the freecad.exe is: https://github.com/donovaly/FreeCADInstProj/releases/download/0.19.3pre/updates_for_LibPack.zip

**Comment by donovaly** [comment]
Fixed with this installer for FC 0.19.4: https://github.com/FreeCAD/FreeCAD/releases/download/0.19.4/FreeCAD-0.19.4-WIN-x64-installer-2.exe
